source/AI.cpp

```diff
@@ -1836,7 +1836,7 @@ void AI::DoCloak(Ship &ship, Command &command)
 		
 		// If this ship has started cloaking, it must get at least 40% repaired
 		// or 40% farther away before it begins decloaking again.
-		double hysteresis = ship.Cloaking() ? 1.4 : 1.;
+		double hysteresis = ship.IsCloaking() ? 1.4 : 1.;
```

Comment:
I think that would be preferable, though. That is, if a ship has decided to cloak, it shouldn't stop unless it's done repairing, even if something does interrupt it.